pkg/acsengine/engine.go

```diff
@@ -984,6 +987,27 @@ func getDataDisks(a *api.AgentPoolProfile) string {
 	return buf.String()
 }
 
+func getEtcdDisk(orchestratorType api.OrchestratorType, m *api.MasterProfile) string {
+	if orchestratorType != api.Kubernetes {
+		return ""
+	}
+	var buf bytes.Buffer
+	buf.WriteString("\"dataDisks\": [\n")
+	etcdDisks := `            {
+              "createOption": "Empty",
+              "diskSizeGB": "%d",
+              "lun": %d,
+              "name": "[concat(variables('masterVMNamePrefix'), copyIndex(),'-etcddisk')]",
+              "vhd": {
+                "uri": "[concat(reference(concat('Microsoft.Storage/storageAccounts/',variables('masterStorageAccountName')),variables('apiVersionStorage')).primaryEndpoints.blob,'vhds/',    variables('masterVMNamePrefix'),copyIndex(),'-etcddisk.vhd')]"
+               }
+            }`
+
```

Comment:
For lun, add as constant, with comment on why it must be lun0 (https://github.com/anhowe/scratch/tree/master/simplelinux-attachblankdisk#lun-0-issues)